### ISSUE TITLE
"Object reference not set to an instance of an object" error when closing solution.

### ISSUE THREAD

**Issue by dmich2** [codecadwallader/codemaid#181]
CodeMaid v0.8.1
VS2013 
Opening any file in the tiny `FSMenu` F# project of https://github.com/dmich2/VSPackagesInFSharp and then closing the solution with the file open causes an "Object reference not set to an instance of an object" error popup to appear.

* Doesn't happen when I disable CodeMaid.
* Doesn't happen if no files from the `FSMenu` project are open when I close the solution.
* None of the other projects in that repo cause the error to occur. 
* No relevant entries in %AppData%\Microsoft\VisualStudio\12.0\ActivityLog.xml when I start devenv.exe with /log

I don't have VS2015 installed so can't build CodeMaid myself to correct the problem.

**Comment by codecadwallader** [comment]
Thanks for reporting the issue and the details you have provided.  I will take a look.

**Comment by codecadwallader** [comment]
I have been unable to reproduce this using the latest Alpha build.  We did find and correct an issue that would cause a somewhat similar bug with PowerShell and Node.JS projects.  Can you test the latest CodeMaid v0.8.8-alpha and see if the issue is still reproducible?  https://github.com/codecadwallader/codemaid/releases/tag/v0.8.8-alpha

**Comment by dmich2** [comment]
Before trying the alpha, I tried to reproduce the problem. Unfortunately, I have since modified this Solution, and the bug no longer occurs. I downloaded my repo from github and was also unable to reproduce.

Next time I will just make a copy of the entire solution instead of relying on source code control. `VSPackagesInFSharp.v12.suo` and my `.hg` folder were ignored by my `.gitignore` file so that may explain the changed behavior.
